content/en/profiler/enabling/ruby.md

```diff
@@ -51,7 +51,7 @@ To begin profiling applications:
 2. Add the `ddtrace` gem to your `Gemfile` or `gems.rb` file:
 
     ```ruby
-    gem 'ddtrace', '~> 1.15'
+    gem 'ddtrace', '~> 1.20'
```

Comment:
Should this not reflect `1.21`? The version in which this change is introduced?

Reply:
Fixed in https://github.com/DataDog/documentation/pull/22037/commits/5cb29c05195ef3813bba130e385fe19c69cb17cf